test/unit/kubernetes-deploy/options_helper_test.rb

```diff
@@ -5,32 +5,32 @@
 class OptionsHelperTest < KubernetesDeploy::TestCase
```

Comment:
To be exhaustive, I think we're missing test cases for:
- More than one dir
- More than one dir, and one of them is stdin
- More than one stdin specified

Reply:
Done